roles/pulibrary.nginxplus/ADDHOSTS.md

```diff
@@ -0,0 +1,39 @@
+Adding new hosts to Nginx Plus
+==============================
+
+* You will need to register the new Virtual Server at
+  [https://networkregistration.princeton.edu](https://networkregistration.princeton.edu) - for now this is a bug because permission/authority is required from OIT, so
+ping the #Operations Slack channel. The turn around is instant + time for DNS to
```

Comment:
This statement is a bit confusing to me.  I believe this is two steps 1. register at https://networkregistration.princeton.edu and 2. ping the library slack channel.  I assume we will always have to register the server?  Maybe it would help if there was an example of the information you wanted in the slack channel.